### PR TITLE
S03 us70 rules

### PR DESCRIPTION
**Pull request by Kileana27**

### CODE REVIEW

**Review by AnassAmh — APPROVED** [review]
Parfait ! Rien à changer pour ma part.

File: src/app/components/header-mobile/header-mobile.component.html [hunk 1]
```diff
@@ -32,20 +32,20 @@
   @if(isMobileSideNavOpen){
     <div class="sidenav">
         <ul>
-            <a routerLink="home"><li>Création De Personnage</li></a>
+            <a routerLink="accueil"><li>Création De Personnage</li></a>
```
Comment (Benja-san) [inline src/app/components/header-mobile/header-mobile.component.html:35]: a tags inside li and not the opposite

File: src/app/components/header/header.component.html [hunk 2]
```diff
@@ -2,24 +2,24 @@
...
       </div>
       <div class="navbar-center">
         <div class="navbar-left">
           <ul>
             <li class="game-button">
               Jeu
               <ul class="submenu">
-                <a routerLink="home"><li>Introduction</li></a>
-                <a routerLink="home"><li>Règles</li></a>
+                <a routerLink="regles"><li>Introduction</li></a>
```
Comment (Benja-san) [inline src/app/components/header/header.component.html:13]: a tag inside li and not the opposite

File: src/app/components/header/header.component.html [hunk 3]
```diff
@@ -2,24 +2,24 @@
...
+                <a (click)="navigateToSection('reglesdujeu')"><li>Règles</li></a>
```
Comment (Benja-san) [inline src/app/components/header/header.component.html:14]: why don't you use a router link ?

File: src/app/components/header-mobile/header-mobile.component.html [hunk 4]
```diff
@@ -32,20 +32,20 @@
...
             <li>
               Jeu
               <ul class="submenu">
-                <a routerLink="home"><li>Introduction</li></a>
-                <a routerLink="home"><li>Règles</li></a>
+                <a routerLink="regles"><li>Introduction</li></a>
+                <a (click)="navigateToSection('reglesdujeu')"><li>Règles</li></a>
```
Comment (Benja-san) [inline src/app/components/header-mobile/header-mobile.component.html:40]: why don't you use a router link ?

File: src/app/pages/home-page/home-page.component.scss [hunk 5]
```diff
@@ -4,8 +4,9 @@
     background-image: url("../../../assets/background_pictures/home_page_background.png");
     background-size: cover;
     background-position: center center;
-    width: 100vw;
-    height: 100vh;
+    //width: 100vw;
```
Comment (Benja-san) [inline src/app/pages/home-page/home-page.component.scss:7]: remove dead code

File: src/app/pages/rules-page/rules-page.component.ts [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+  imports: [],
+  templateUrl: './rules-page.component.html',
+  styleUrl: './rules-page.component.scss'
+})
+export class RulesPageComponent {
+
+  constructor(private route: ActivatedRoute) {}
+
+  ngOnInit() {
+    this.route.fragment.subscribe((fragment: string|null) => {
```
Comment (Benja-san) [inline src/app/pages/rules-page/rules-page.component.ts:16]: need a talk about those lines

**Review by Benja-san — CHANGES_REQUESTED** [review]

File: src/app/components/header-mobile/header-mobile.component.html (hunk 1 above)
Comment (Kileana27) [reply src/app/components/header-mobile/header-mobile.component.html:35]: Not my us but corrected it

File: src/app/components/header/header.component.html (hunk 2 above)
Comment (Kileana27) [reply src/app/components/header/header.component.html:13]: Not my us but corrected it

File: src/app/components/header/header.component.html (hunk 3 above)
Comment (Kileana27) [reply src/app/components/header/header.component.html:14]: Now using a router link thanks to you !

File: src/app/components/header-mobile/header-mobile.component.html (hunk 4 above)
Comment (Kileana27) [reply src/app/components/header-mobile/header-mobile.component.html:40]: Now using a router link thanks to you !

File: src/app/pages/home-page/home-page.component.scss (hunk 5 above)
Comment (Kileana27) [reply src/app/pages/home-page/home-page.component.scss:7]: Removed

**Review by Benja-san — APPROVED** [review]

**Pull request merged**